test/cljs/status_im/test/i18n.cljs

```diff
@@ -1,6 +1,27 @@
 (ns status-im.test.i18n
   (:require [cljs.test :refer-macros [deftest is]]
-            [status-im.i18n :as i18n]))
+            [status-im.i18n :as i18n]
+            [clojure.set :as set]
+            [cljs.spec.alpha :as s]))
 
 (deftest label-options
   (is (not (nil? (:key (i18n/label-options {:key nil}))))))
+
+(deftest locales-only-have-existing-tran-ids
+  (is (s/valid? ::i18n/trans-ids (i18n/trans-ids-for-all-locales))
+      (->> i18n/locales
+           (remove #(s/valid? ::i18n/trans-ids (i18n/locale->trans-ids %)))
+           (map (fn [l]
```

Comment:
Can we break this out into a separate fn perhaps? To ensure line stays within 80 chars for readability